stanford/roles/cluster/defaults/main.yml

```diff
@@ -76,7 +76,7 @@ CLUSTER_SUBNETS:
       cidr: "{{ DEPLOYMENT_CIDR_BASE }}.55.0/24"
     - zone: 'us-west-1c'
       cidr: "{{ DEPLOYMENT_CIDR_BASE }}.56.0/24"
-CLUSTER_NAT_SUBNET: "{{ CLUSTER_SUBNETS['nat'][cluster_subnet_index|int] }}"
+CLUSTER_NAT_SUBNET: "{{ CLUSTER_SUBNETS['public'][cluster_subnet_index|int] }}"
```

Comment:
Should `CLUSTER_NAT_SUBNET` also be renamed to `CLUSTER_PUBLIC_SUBNET`?

Reply:
Ah, probably.

Reply:
Actually, no.
Where it's used in the play, it's explicitly for creating the NAT box/subnet, so it's more just saying "the nat subnet should use the public subnet"